src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +71,28 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameOcrFieldName(String formType) {
+        return formTypeToSurnameOcrFieldMappings.get(formType);
+    }
+
+    public void setFormTypeToSurnameOcrFieldMappings(List<FormFieldMapping> formTypeToSurnameOcrFieldMappings) {
+        this.formTypeToSurnameOcrFieldMappings = formTypeToSurnameOcrFieldMappings
+            .stream()
+            .collect(
+                HashMap::new,
+                (map, mapping) -> {
+                    String formType = mapping.getFormType();
+                    if (map.containsKey(formType)) {
+                        String msg = String.format("Form type %s has multiple mappings to surname fields.", formType);
+                        log.error(msg);
+                        throw new InvalidConfigurationException(msg);
+                    } else {
+                        map.put(formType, mapping.getOcrField());
+                    }
+                },
+                HashMap::putAll
+            );
+    }
```

Comment:
Suggestion: we can now reduce it to something simpler - making a map using Collections and throwing an exception if there's any problem. Up to you.

Reply:
Not 100% sure but I think spring automatically creates map when only 2 yaml list entries are present. A quick test-try will prove such wonder